.gitignore

```diff
@@ -198,3 +198,6 @@ tests/fsyacc/test1compat.exe.config
 tests/fsyacc/test2.exe.config
 tests/fsyacc/test2compat.exe.config
 .paket
+
+.idea/
+.vs/
```

Comment:
@teo-tsirpanis  can you please revert this in PR? this change already merged into master